modules/gapi/samples/pipeline_modeling_tool.cpp

```diff
@@ -224,6 +224,7 @@ int main(int argc, char* argv[]) {
                                    " if set to 0. If it's specified will be"
                                    " applied for every pipeline. }"
         "{ app_mode    | realtime  | Application mode (realtime/benchmark). }"
+        "{ drop_frames | false     | Drop frames if it comes faster than pipeline is completed. }"
```

Comment:
if `frames` is plural, it should be `come` not `comes`. I'd also recommend using `earlier` rather than `faster` in this case. Please rephrase.

Reply:
Done